Shared/Flows/Onboarding/Welcome/WelcomeViewController.swift

```diff
@@ -174,26 +174,19 @@ class WelcomeViewController: DiffableCollectionViewController<MessageSequenceSec
 
             // Put Benji's messages at the top, and all other messages below.
             var benjiMessages: [MessageSequenceItem] = []
-            var otherMessages: [MessageSequenceItem] = []
 
             let allMessages = conversationController.messages.filter { message in
                 return !message.isDeleted
             }
 
             allMessages.forEach({ message in
-                if message.authorId == PFConfig.current().adminUserId {
-                    benjiMessages.append(MessageSequenceItem.message(cid: cid,
-                                                                     messageID: message.id,
-                                                                     showDetail: false))
-                } else {
-                    otherMessages.append(MessageSequenceItem.message(cid: cid, messageID:
-                                                                        message.id,
-                                                                     showDetail: false))
-                }
+                guard message.authorId == PFConfig.current().adminUserId else { return }
+                benjiMessages.append(MessageSequenceItem.message(cid: cid,
+                                                                 messageID: message.id,
+                                                                 showDetail: false))
```

Comment:
If there's only one stack, I assume we'll just show the Benji message for now?

Reply:
No we will show both. But this may no longer work... We can look at this later.